gptel.el

```diff
@@ -374,7 +374,8 @@ transient menu interface provided by `gptel-menu'."
   '((default . "You are a large language model living in Emacs and a helpful assistant. Respond concisely.")
     (programming . "You are a large language model and a careful programmer. Provide code and only code as output without any additional text, prompt or note.")
     (writing . "You are a large language model and a writing assistant. Respond concisely.")
-    (chat . "You are a large language model and a conversation partner. Respond concisely."))
+    (chat . "You are a large language model and a conversation partner. Respond concisely.")
+    (none))
```

Comment:
Why is `none` an option here?  There appears to be no way to actually choose this to select `nil` as the value of `gptel--system-message`.